### PR TITLE
feat(open_in_browser): Add open in browser to different screens

### LINKED ISSUE #285: Open in browser feature
It would to nice to have a button to open safari to perform anythings gitpoint is not (yet) able to perform, e.g. editing comment and look at commit history.

### PR DESCRIPTION
**Pull request by jouderianjr**
Screens;
  - Issue
  - Repository
  - User Profile
  - Organization
  - Auth User Profile
  - README

Resolves #285

### CODE REVIEW

File: src/locale/languages/en.js [hunk 1]
```diff
@@ -229,12 +229,16 @@ export const en = {
       deleted: 'DELETED',
       fileRenamed: 'File renamed without any changes',
     },
+    readMe: {
+      readMeActions: 'README actions',
```
Comment (andrewda) [inline src/locale/languages/en.js]: Not sure, but maybe capitalize the "A" in "actions"? CC @housseindjirdeh

File: src/locale/languages/en.js [hunk 2]
```diff
@@ -334,5 +338,6 @@ export const en = {
     abbreviations: {
       thousand: 'k',
     },
+    openInBrowser: 'Open in browser',
```
Comment (andrewda) [inline src/locale/languages/en.js]: Maybe capitalize this, too? "Open in Browser"

File: src/locale/languages/fr.js [hunk 3]
```diff
@@ -226,12 +226,16 @@ export const fr = {
       deleted: 'SUPPRIMÉ',
       fileRenamed: 'Fichier renommé sans aucune modification',
     },
+    readMe: {
+      readMeActions: 'README actions',
```
Comment (andrewda) [inline src/locale/languages/fr.js]: "Actions sur le README"

File: src/locale/languages/fr.js [hunk 4]
```diff
@@ -226,12 +226,16 @@ export const fr = {
...
+    },
   },
   organization: {
     main: {
       membersTitle: 'MEMBRES',
       descriptionTitle: 'DESCRIPTION',
     },
+    organizationActions: 'Organization Actions',
```
Comment (andrewda) [inline src/locale/languages/fr.js]: "Actions sur l'organization"

File: src/locale/languages/fr.js [hunk 5]
```diff
@@ -331,5 +335,6 @@ export const fr = {
     abbreviations: {
       thousand: 'k',
     },
+    openInBrowser: 'Open in browser',
```
Comment (andrewda) [inline src/locale/languages/fr.js]: "Ouvrir dans le navigateur"

File: src/components/repository-list-item.component.js [hunk 6]
```diff
@@ -87,7 +88,7 @@ const renderTitle = (repository, showFullName) =>
 
       <Icon
         containerStyle={styles.extraInfoIcon}
-        name="repo-forked"
+        name="ellipsis"
```
Comment (chinesedfan) [inline src/components/repository-list-item.component.js]: @jouderianjr  Hi, can you explain why these 2 icons are modified?

Comment (jouderianjr) [reply src/components/repository-list-item.component.js]: My bad, I was testing the vector-icon issue and I added in the commit by mistake. Nice catch! 🙈

**Comment by jouderianjr** [comment]
done @andrewda. Thanks! 💃

File: src/locale/languages/ru.js [hunk 7]
```diff
@@ -227,12 +227,16 @@ export const ru = {
...
+    readMe: {
+      readMeActions: 'README Actions',
+    },
   },
   organization: {
     main: {
       membersTitle: 'УЧАСТНИКИ',
       descriptionTitle: 'ОПИСАНИЕ',
     },
+    organizationActions: 'Organization Actions',
```
Comment (lex111) [inline src/locale/languages/ru.js]: "Действия с организацией"

File: src/locale/languages/ru.js [hunk 8]
```diff
@@ -227,12 +227,16 @@ export const ru = {
       deleted: 'УДАЛЕННЫЙ',
       fileRenamed: 'Файл переименован без каких-либо изменений',
     },
+    readMe: {
+      readMeActions: 'README Actions',
```
Comment (lex111) [inline src/locale/languages/ru.js]: "Действия с README"

File: src/locale/languages/ru.js [hunk 9]
```diff
@@ -332,5 +336,6 @@ export const ru = {
     abbreviations: {
       thousand: ' тыс.',
     },
+    openInBrowser: 'Open in browser',
```
Comment (lex111) [inline src/locale/languages/ru.js]: "Открыть в браузере"

**Comment by jouderianjr** [comment]
Done @lex111, thank you for the help 😄

**Comment by lex111** [comment]
> Issue
> ListItem in the settings screen.

@jouderianjr Hmm, probably because of this I do not see this action, because I have no permissions to the issue. And I do not see the icons with the settings, and therefore there is no way to open the issue in the browser. Can we add ActionSheet then, and we will show there instead of show this action in the settings screen?

File: src/repository/screens/read-me.screen.js [hunk 10]
```diff
@@ -33,19 +37,40 @@ const styles = StyleSheet.create({
 });
 
 class ReadMe extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { state } = navigation;
+
+    return {
+      headerRight: (
+        <Icon
+          name="gear"
```
Comment (lex111) [inline src/repository/screens/read-me.screen.js]: Can this icon be better here?

image

Comment (jouderianjr) [reply src/repository/screens/read-me.screen.js]: yeah, using ellipsis is more consistent, I will change it

**Comment by jouderianjr** [comment]
> @jouderianjr Hmm, probably because of this I do not see this action, because I have no permissions to the issue. And I do not see the icons with the settings, and therefore there is no way to open the issue in the browser. Can we add ActionSheet then, and we will show there instead of show this action in the settings screen?

Humm, I got it, but where we will add the button to open the ActionSheet? Maybe if the user doesn't have the permission the button will pop up an action sheet, that is what you are proposing? 😄

**Comment by lex111** [comment]
@jouderianjr Yes, that's right, there will be another "ellipsis" icon, and the action sheet will open, similarly as on the README screen.

**Comment by jouderianjr** [comment]
@lex111 I've added the ellipsis icon, please test it 💃

File: src/repository/screens/read-me.screen.js (hunk 10 above)
Comment (lex111) [reply src/repository/screens/read-me.screen.js]: On the README screen still use the ellipsis icon.

File: src/locale/languages/ru.js [hunk 11]
```diff
@@ -274,6 +278,7 @@ export const ru = {
         '#{{number}} открыт {{time}} назад пользователем {{user}}',
       closedIssueSubTitle:
         '#{{number}} закрыт {{time}} назад пользователем {{user}}',
+      issueActions: 'Issue Actions',
```
Comment (lex111) [inline src/locale/languages/ru.js]: "Действия с задачей"

File: src/issue/screens/issue.screen.js [hunk 12]
```diff
@@ -78,11 +79,22 @@ class Issue extends Component {
...
               })}
           />
-        ),
-      };
-    }
+        );
+      }
+
+      return (
+        <Icon
```
Comment (lex111) [inline src/issue/screens/issue.screen.js]: It seems that when press it, there should be a change in the transparency of the icon, but it's not now, can we add?

Comment (jouderianjr) [reply src/issue/screens/issue.screen.js]: Hey @lex111, I'm using this icon like have in other places. I think all the right icons in the app has `underlayColor` transparent. That being said, I think if we would like to add this, we should handle in another PR. What do you think about? 😄

File: src/repository/screens/read-me.screen.js (hunk 10 above)
Comment (jouderianjr) [reply src/repository/screens/read-me.screen.js]: Ops, my bad, sorry 🙈

File: src/issue/screens/issue.screen.js (hunk 12 above)
Comment (lex111) [reply src/issue/screens/issue.screen.js]: @jouderianjr ok, it's just weird that it's not done now for README and issue screens.

**Pull request merged**